src/std/io/file.ss

```diff
@@ -71,6 +71,15 @@
         (set! self.closed? #t)
         (close-port self.fd)))))
 
+(defmethod {seek file-io}
+  (lambda (self position from: (from 'start))
```

Comment:
No need for any checks in the method implementation, this is handled by the interface.
Also, no keywords please, just a plain argument that is optional at the interface level; no need to be optional in the method implemenation, it is already resolved at the interface.